src/types.rs

```diff
@@ -600,14 +609,58 @@ pub struct ContractAddress {
     serde(tag = "type", content = "address", rename_all = "lowercase")
 )]
 #[cfg_attr(feature = "fuzz", derive(Arbitrary))]
-#[derive(PartialEq, Eq, Copy, Clone, Debug)]
+#[derive(Eq, Copy, Clone, Debug)]
 pub enum Address {
     Account(AccountAddress),
     Contract(ContractAddress),
 }
 
+// This trait is implemented manually to save bytes
```

Comment:
Maybe rephrase this to something like 
"The trait is implemented manually so that the generated Wasm code is smaller."